app/views/books/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>New Book</h1>
+<%= form_for @book do |f| %>
+  <% if flash[:notice] %>
+    <p><%= flash[:notice] %></p>
+  <% end %>
+  <%= f.label :title%>
+  <%= f.text_field :title, type: :name, required: true %>
+  <br>
+  <%= f.label :year_published %>
+  <%= f.text_field :year_published, type: :number, min: -4000, max: 2019, required: true%>
```

Comment:
Same issue as the model itself - either don't add a max for the year or do an ERB block to do `Time.now.year`.